obs_maven/core.py

```diff
@@ -31,7 +31,7 @@
 
 
 class Configuration:
-    def __init__(self, config_path, repo):
+    def __init__(self, config_path, repo, cache_path):
```

Comment:
nit: I'd consider declaring `Configuration.cache_path` as a static variable to avoid lengthy arguments list passing.

Reply:
Sure, I would prefer that, but was not sure about the best practice here.

Reply:
Leaving it unchanged following recommendations for better testability.